target/linux/generic/pending-5.4/780-net-dsa-allow-for-multiple-CPU-ports.patch

```diff
@@ -0,0 +1,212 @@
+From 3cbd1b83e555e33448f78cbdae3a0b691c8419cf Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Marek=20Beh=C3=BAn?= <marek.behun@nic.cz>
+Date: Tue, 5 May 2020 19:22:58 +0200
+Subject: [PATCH] net: dsa: allow for multiple CPU ports
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Allow for multiple CPU ports in a DSA switch tree. By default assign the
+CPU ports to user ports in a round robin way, ie. if there are two CPU
```

Comment:
I prefer assign one to WAN port, the other to LAN ports by default.

Reply:
That only makes sense for the Linksys devices. The Turris Omnia's WAN port is not part of the switch.